src/analytics.ts

```diff
@@ -19,12 +19,33 @@ export function createAnalyticsInstance(options?: Options) {
         event_cache: Array<{ event: keyof TEvents; payload: TEvents[keyof TEvents] }> = [],
         page_view_cache: Array<{ current_page: string; platform: string; user_id: string }> = []
 
-    const initialise = ({ growthbookKey, growthbookDecryptionKey, rudderstackKey, growthbookOptions }: Options) => {
+    const initialise = ({
+        growthbookKey,
+        growthbookDecryptionKey,
+        rudderstackKey,
+        GBAttributes = {},
```

Comment:
[Q] isn't attributes is also [part of options](https://docs.growthbook.io/lib/react#attributes)? `options.attributes`, so instead of passing it separately, consumer can pass from options too, and handle it from there

Reply:
Done